src/video/dto/video.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiProperty, ApiPropertyOptional, PartialType } from '@nestjs/swagger';
+import { LocaleText } from 'src/common/dto/locale-text.dto';
+
+export class CreateVideoDto {
+  @ApiProperty({ type: LocaleText })
+  title: LocaleText;
+
+  @ApiProperty({ example: 'MUSIC_VIDEO', enum: ['MUSIC_VIDEO', 'VLOG', 'LIVE', 'BEHIND', 'ETC'] })
+  type: string;
+
+  @ApiProperty({ example: 'https://youtube.com/watch?v=123' })
+  externalUrl: string;
+
+  @ApiPropertyOptional({ example: 'https://cdn.hebi.im/thumbs/video1.jpg' })
```

Comment:
```suggestion
  @ApiPropertyOptional({ example: 'https://static.hebi.im/thumbs/video1.jpg' })
```

Also, for user-uploaded (means by API) will be uploaded to user-content.hebi.im, so maybe we could say:
```suggestion
  @ApiPropertyOptional({ example: 'https://user-content.hebi.im/www/thumbs/video1.jpg' })
```